packages/web/package.json

```diff
@@ -30,7 +30,7 @@
     "prepublishOnly": "yarn cross-env NODE_ENV=production yarn build",
     "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\"",
     "build:types": "ttsc --build --verbose",
-    "build:watch": "nodemon --watch src --ext \"js,ts,tsx\" --ignore dist --exec \"yarn build\"",
+    "build:watch": "nodemon --watch src redwood.toml --ext \"js,ts,tsx,toml\" --ignore dist --exec \"yarn build\"",
```

Comment:
This should be reverted since it doesn't do anything, there is no redwood.toml file here.

Reply:
I think what Peter is saying is that `redwood.toml` isn't in `src` but I think that you may have the `--watch` config wrong.

Have a look at the docs here: https://github.com/remy/nodemon#monitoring-multiple-directories

> Don't use unix globbing to pass multiple directories, e.g --watch ./lib/*, it won't work. You need a --watch flag per directory watched.

So maybe:

```
"build:watch": "nodemon --watch redwood.toml --watch src --ext \"js,ts,tsx,toml\" --ignore dist --exec \"yarn build\"",
```

?